commands/run/config/samples.ini

```diff
@@ -0,0 +1,57 @@
+[stub]
```

Comment:
Sample definitions. Simplified from before, as much now is defined in the csv templates. Sex, sample type and input datasets remains